package/piksi_system_daemon/src/main.c

```diff
@@ -199,18 +199,51 @@ static int date_string_get(const char *timestamp_string,
 static void img_tbl_settings_setup(settings_ctx_t *settings_ctx)
 {
   char name_string[64];
+  char uimage_string[64];
```

Comment:
A couple small style things.

It seems like it would be clearer to define the string buffer size and use that instead of hard coding a couple strings then using sizeof. ex:

```
#define STR_BUFFER_SIZE 64
char name_string[STR_BUFFER_SIZE];
char uimage_string[STR_BUFFER_SIZE];
...
 if (file_read_string("/img_tbl/boot/name",
                        name_string, STR_BUFFER_SIZE) == 0) {
...
static char imageset_build_id[STR_BUFFER_SIZE];
``

Reply:
that would be better; willco